aiven_mysql_migrate/config.py

```diff
@@ -11,3 +11,7 @@
 SOURCE_SERVICE_URI = os.getenv("SOURCE_SERVICE_URI")
 TARGET_SERVICE_URI = os.getenv("TARGET_SERVICE_URI")
 TARGET_MASTER_SERVICE_URI = os.getenv("TARGET_MASTER_SERVICE_URI")
+
+SOURCE_SSL_CA = os.getenv("SOURCE_SSL_CA")
```

Comment:
Naming nitpicking: whenever I see a name like that, I'm always confused about this being a dumped certificate in PEM format, or a path. I can imagine this is a path; would you mind calling this SOURCE_SSL_CA_PATH? Also, if the driver only supports either DER or PEM (should be checked), I think the best name would be SOURCE_SSL_CA_PEM_PATH  or something like that.

Reply:
I'm using these names just to be consistent with mysql cli options...
```
--ssl-ca
--ssl-cert
--ssl-key
```
https://dev.mysql.com/doc/refman/8.0/en/connection-options.html